testsuite/features/step_definitions/lock_packages_on_client.rb

```diff
@@ -3,21 +3,22 @@
 
 Then(/^"(.*?)" is locked on this client$/) do |pkg|
   zypp_lock_file = '/etc/zypp/locks'
-  raise unless file_exists?($client, zypp_lock_file)
+  raise "File already exist: #{zypp_lock_file}" unless file_exists?($client, zypp_lock_file)
```

Comment:
or `File #{zypp_lock_file} already exist` to align with your other error messages

Reply:
it's fine